apps/prairielearn/src/tests/lti13.test.ts

```diff
@@ -463,7 +463,7 @@ describe('LTI 1.3', () => {
       res.send(
         JSON.stringify({
           access_token: ACCESS_TOKEN,
-          stenotype: 'bearer',
+          token_type: 'bearer',
```

Comment:
The newer libraries caught that we weren't including a `token_type` in our testing. I couldn't find a canonical spec for `stenotype` other than in an LTI spec example, so it seemed like a safe change.